ceph-salt-formula/salt/ceph-salt/apply/cephbootstrap.sls

```diff
@@ -6,6 +6,22 @@
 
 {% set bootstrap_ceph_conf = pillar['ceph-salt'].get('bootstrap_ceph_conf', {}) %}
 
+create static bootstrap yaml:
+  cmd.run:
+    - name: |
+        echo -en "" > /tmp/static_bootstrap.yaml
+        echo -en "service_type: mgr\n" >> /tmp/static_bootstrap.yaml
+        echo -en "service_name: mgr\n" >> /tmp/static_bootstrap.yaml
+        echo -en "placement:\n" >> /tmp/static_bootstrap.yaml
+        echo -en "    hosts:\n" >> /tmp/static_bootstrap.yaml
+        echo -en "        - {{ pillar['ceph-salt']['bootstrap_node_hostname'] }}\n" >> /tmp/static_bootstrap.yaml
```

Comment:
Instead of adding a new pillar item, can we simply read `host` grain like we are doing in https://github.com/ceph/ceph-salt/blob/master/ceph-salt-formula/salt/ceph-salt/apply/cephorch.sls#L8 ?

Note that this is running in the bootstrap minion, so it should work.